pkg/client/build.go

```diff
@@ -335,6 +345,11 @@ func (c *Client) Build(ctx context.Context, opts BuildOptions) error {
 		return errors.Wrapf(err, "getting builder architecture")
 	}
 
+	platformToUse := opts.Platform
+	if platformToUse == "" {
+		platformToUse = fmt.Sprintf("%s/%s", builderOS, builderArch) // FIXME: what about arch variant, etc.
```

Comment:
This question existed before this PR I am just wondering what we plan to do about it

Reply:
Fixed in 863eb981